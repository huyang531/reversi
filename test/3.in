cd
cc
ef
ce
bb
cb
db
aa
be
af
ab
ac
bd
bc
ae
ad
da
ca
ba
bf
ag
ea
eb
dc
fb
cf
bg
cg
dg
eh
eg
dh
ah
ec
fg
gh
gg
gf
fe
df
hg
he
ch
ge
fh
bh
gd
hc
gc
hf
hd
hh
fd
hb
gb
ff
fa
ga
ha
fc
