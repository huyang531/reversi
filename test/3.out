  abcdefgh
a ........
b ........
c ........
d ...WB...
e ...BW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ...B....
d ...BB...
e ...BW...
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WB....
d ...WB...
e ...BW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WB....
d ...WB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ...WB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b .B......
c ..BWW...
d ...BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b .B......
c .WWWW...
d ...BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b .B......
c .BWWW...
d .B.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a W.......
b .W......
c .BWWW...
d .B.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a W.......
b .W..B...
c .BWWB...
d .B.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a W....W..
b .W..W...
c .BWWB...
d .B.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WB...W..
b .B..W...
c .BWWB...
d .B.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWW..W..
b .B..W...
c .BWWB...
d .B.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWW..W..
b .B.BW...
c .BBBB...
d .B.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWW..W..
b .BWWW...
c .BBBB...
d .B.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWW.BW..
b .BWBB...
c .BBBB...
d .B.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWW..
b .BWBB...
c .BBBB...
d .B.BB...
e ...BBB..
f ........
g ........
h ........
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWW..
b .BWBB...
c .WBBB...
d WB.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWW..
b .BWBB...
c BBBBB...
d WB.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWW..
b WWWBB...
c WBBBB...
d WB.BB...
e ...BBB..
f ........
g ........
h ........
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWW..
b WWWWWW..
c WBBBB...
d WB.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWB..
c WBBBB...
d WB.BB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWB..
c WBWBB...
d WW.BB...
e W..BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWB..
c WBWBB...
d WB.BB...
e WB.BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWB..
c WWWWB...
d WWWBB...
e WB.BBB..
f ........
g ........
h ........
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWB..
c WWWWB...
d WWWBB...
e WW.BBB..
f .W......
g ........
h ........
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWW..
c WWWWWW..
d WWWBB...
e WW.BBB..
f .W......
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWWB.
c WWWWWB..
d WWWBB...
e WW.BBB..
f .W......
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWWB.
c WWWWWWW.
d WWWBB...
e WW.BBB..
f .W......
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWWB.
c WWWWWWB.
d WWWBB.B.
e WW.BBB..
f .W......
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWWB.
c WWWWWWB.
d WWWBB.W.
e WW.BBB.W
f .W......
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWWB.
c WWWWWWB.
d WWWBB.B.
e WW.BBBBW
f .W......
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWB.
b WWWWWWB.
c WWWWWWW.
d WWWBB.BW
e WW.BBBBW
f .W......
g ........
h ........
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWW.
d WWWBB.BW
e WW.BBBBW
f .W......
g ........
h ........
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWW.
d WWWWB.BW
e WWWWWWWW
f .W......
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWW.
d WWWWB.BW
e WWWWWBBW
f .W....B.
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWW.
d WWWWW.BW
e WWWWWWBW
f .W....W.
g .......W
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWW.
d WWWWW.BW
e WWWWWWBW
f .W....B.
g ......BW
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWW.
d WWWWW.BW
e WWWWWWBW
f .W....W.
g .....WWW
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWW.
d WWWWW.BW
e WWWWWBBW
f .W..B.W.
g .....WWW
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWW.
d WWWWWWWW
e WWWWWBBW
f .W..B.W.
g .....WWW
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWW.
d WWWWWWWW
e WWWWWBBW
f .W..B.B.
g .....BBW
h ......B.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWW.
d WWWWWWWW
e WWWWWBBW
f .W..B.W.
g .....WBW
h ....W.B.
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWWB
d WWWWWWBW
e WWWWWBBW
f .W..B.W.
g .....WBW
h ....W.B.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWWB
d WWWWWWBW
e WWWWWBBW
f .W..W.W.
g ....WWBW
h ....W.B.
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWW.
c WWWWWWWB
d WWWWWWBB
e WWWWWBBB
f .W..W.WB
g ....WWBW
h ....W.B.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWBW
e WWWWWBBW
f .W..W.WW
g ....WWBW
h ....W.B.
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWBW
e WWWWWBBW
f .W..B.WW
g ...BBBBW
h ....W.B.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWBW
f .W..W.WW
g ...WBBBW
h ..W.W.B.
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWBW
f .W..W.WW
g ..BBBBBW
h ..W.W.B.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWBW
f .W..W.WW
g ..BBBBWW
h ..W.WWB.
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWBW
f .W..W.WW
g ..BBBBWW
h ..WBBBB.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWBW
f .W..W.WW
g ..BBBBWW
h ..WWWWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWBW
f .W.WW.WW
g ..BWWBWW
h ..WWWWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWBW
f .W.WW.WW
g ..WWWBWW
h .WWWWWWW
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWBW
f .W.WW.WW
g .BBBBBWW
h .WWWWWWW
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f .W.WWWWW
g .BBBWWWW
h .WWWWWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f WW.WWWWW
g .WBBWWWW
h .WWWWWWW
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f WW.WWWWW
g BBBBWWWW
h .WWWWWWW
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f WW.WWWWW
g WBBBWWWW
h WWWWWWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f WWWWWWWW
g WWWWWWWW
h WWWWWWWW
B player has no valid move.
W player has no valid move.
White wins by 64 points!
